### PR TITLE
all: update maintainer and author information.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

### CODE REVIEW

**Review by shaun-edwards — DISMISSED** [review]

File: ur5_moveit_config/package.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <description>
      An automatically generated package with all the configuration and launch files for using the ur5 with the MoveIt Motion Planning Framework
   </description>
-  <author email="fxm@ipa.fhg.de">Felix Messmer</author>
+  <author>Felix Messmer</author>
   <maintainer email="fxm@ipa.fhg.de">Felix Messmer</maintainer>
```
Comment (ipa-nhg) [inline ur5_moveit_config/package.xml]: any special reason to keep the maintainer  only for `ur5_moveit_config` ?

Comment (gavanderhoorn) [reply ur5_moveit_config/package.xml]: No, seems I missed that one.

Should be fixed now in fce99c1.

**Comment by ipa-nhg** [comment]
@fmessmer ok for you? now that [ur_modern_driver](https://github.com/ros-industrial/ur_modern_driver) is supported by ROS-I we will have to refactor some code here and this require extra maintership efforts

**Comment by fmessmer** [comment]
fine for me...I got kicked from the push group anyway already

**Review by fmessmer — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
> fine for me...I got kicked from the push group anyway already

@fmessmer: that was me, and was a bit unceremonial unfortunately. More so than I'd liked. This was part of a larger cleanup of the push and admin groups that included the ones for `universal_robot`.

I'd like to thank you for all the work that you've done while maintaining and developing these packages and I can safely say that you've kept a very popular set of packages in good condition for quite some time, providing the ROS community a great service.

I wish you success at mojin-robotics and I hope we can still reach out to you if/when we run into things you've dealt with in the past.

**Review by ipa-nhg — APPROVED** [review]

**Comment by fmessmer** [comment]
@gavanderhoorn thanks for the kind words
sure thing...I'll stay responsive as much as I can...:laughing:

**Pull request merged**